docs/source/examples/dispatch_tutorial.ipynb

```diff
@@ -0,0 +1,522 @@
+    "\n",
+    "2. A list of `osier.Technology` objects that can meet the specified demand.\n",
+    "\n",
+    "In order to solve the model, simply call `DispatchModel.solve(solver=\"SOLVER_NAME\")`. This function returns `None`, but you can check that it succeeded by printing `DispatchModel.objective`.\n",
+    "\n",
+    "The default values for the model are:\n",
+    "\n",
+    "1. Energy -- Megawatts [MW]\n",
+    "\n",
+    "2. Time -- 1 hour [hr]\n",
+    "\n",
+    "3. Cost -- million dollars per unit [M$/unit]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.0041415950000000005\n",
+      "Model ran in 0.987957 seconds.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import time\n",
+    "start = time.perf_counter()\n",
+    "model = DispatchModel(technology_list=technology_mix,\n",
+    "                      net_demand=demand\n",
+    "                     )\n",
+    "model.solve(solver='glpk')  # add your preferred solver here!\n",
+    "end = time.perf_counter()\n",
+    "print(model.objective)\n",
+    "print(f\"Model ran in {(end-start):3f} seconds.\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Verifying the results\n",
+    "\n",
+    "Since this is only a dispatch model, the only costs that contribute to the merit order is the marginal cost of producing electricity (i.e., the variable costs). Although there are two types of variable costs in an `osier.Technology` object, fuel and operating and management (O&M). They are automatically combined into a single variable cost (`osier` tries to enforce consistent units as well by using the `unyt` library).\n",
+    "\n",
+    "In order to verify that the results make sense, we can divide the model objective by the total demand, and that should equal the variable cost for the natural gas technology (though I use the `np.isclose` command to account for any floating point wonkiness)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tol = 1e-6\n",
+    "np.isclose((model.objective / total_demand), \n",
+    "           model.technology_list[0].variable_cost, \n",
+    "           rtol=tol)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The variables agree! Which means the model ran successfully and nothing strange happened!"
```

Comment:
I think this is probably also a little too casual.

Reply:
Why does this matter?